### DISCUSSION TITLE
How do I revert to a stable release?

### DISCUSSION THREAD

**Discussion by denelon** [microsoft/winget-cli#1960] [Q&A] [answered]
From a Twitter request:

Hello there. I have updated Winget manually. Now I want to revert this update. How can I revert to the old stable application installer?

**Comment by denelon** [comment]
https://twitter.com/furcanomer/status/1296862648265474050

**Comment by denelon** [comment]
We are working through a handful of scenarios to validate behavior. This will all be documented once we have verified the results.

**Comment by LSeelig** [comment]
Is there an answer here? The original tweet has been deleted.

**Comment by denelon** [comment]
@LordMethenor we are working through the last step of the process to provide the correct stable version of the "App Installer" without the Windows Package Manager so users can revert to a previous version. The Microsoft Store will still provide the "latest" version available for a given version of Windows 10 or Windows 11 (insider build).

**Comment by denelon** [comment, marked as answer]
The [Windows Package Manager 1.1](https://github.com/microsoft/winget-cli/releases/tag/v1.1.12653) release is available to install from GitHub for users who wish to revert to a stable release.
